style.css

```diff
@@ -0,0 +1,4 @@
+h1 {

```

Comment:
Please use the class of h1 here and not the h1 itself

Reply:
the requested changes has been effected